recipe/meta.yaml

```diff
@@ -27,14 +27,17 @@ requirements:
         - zlib
 
 test:
+    requires:
+        - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
Sounds good. We will definitely try to figure out something better going forward.